packages/esm-ward-app/src/config-schema.ts

```diff
@@ -200,10 +200,16 @@ export const configSchema: ConfigSchema = {
       },
     },
   },
+  admissionEncounterTypeUuid: {
+    _type: Type.UUID,
+    _description: 'The encounter type UUID for admission encountertype',
+    _default: '260566e1-c909-4d61-a96f-c1019291a09d',
+  },
```

Comment:
Instead of adding a config property for this, we should fetch this (and the transfer encounter type) from the EMR API config endpoint: http://localhost:8080/openmrs/ws/rest/v1/emrapi/configuration

Reply:
Oh, and I believe @usamaidrsk  is currently working on something to fetch the emr api, so it might be worth checking in with him to see if there is a shared resource you can use.